include/stronk/unit_v2.h

```diff
@@ -223,4 +257,14 @@ constexpr auto make(UnderlyingT&& value)
     return unit_value_t<UnitT, UnderlyingT> {std::forward<UnderlyingT>(value)};
 }
 
+template<typename ScaleT, unit_like UnitT, typename UnderlyingT>
+constexpr auto make(UnderlyingT&& value)
+{
+    static_assert(UnitT::dimensions_t::size() == 1,
+                  "this function semantically does not seem right with multiple units");
+    static_assert(UnitT::dimensions_t::first_t::rank == 1, "this function is ambiguous for none rank 1");
```

Comment:
Does this prevent me from making a EuroPerMWh if that type was composed of Euro and MWh units?